build/Linux+BSD/portable/RecipeArm

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# This file is designed to run in debian jessie docker images
+
+main() {
+
+  set -e # Halt on error
+  set -x # Be verbose
+
+  case $1 in
+    "--fetch-dependencies-only")
+      fetchDependencies "$@"
+      ;;
+    "--build-only")
+      shift #need to adjust parameter list so $makefile_overrides start at $1
+      build "$@"
+      ;;
+    "--package-only")
+      package "$@"
+      ;;
+    *)
+      fetchDependencies "$@"
+      build "$@"
+      package "$@"
+  esac
+
+  echo 'RecipeARM completed'
+}
+
+##########################################################################
+# The dependences that musescore needs? to run, to include in package
+##########################################################################
+fetchDependencies() {
+  echo 'fetching'
+# not using right now...currently just put all dependencies in each docker file
+# might want to split up between musescore compile dependencies, musescore runtime dependencies, & AppImageKit deps
+
+#  echo `apt-get install -y libqt4-dev libsndfile1-dev libasound2-dev portaudio19-dev zlib1g-dev libfreetype6-dev libssl-dev libdrm-dev libgl1-mesa-dev libpulse-dev qtbase5-dev qttools5-dev libqt5multimedia5 libqt5multimediawidgets5 libqt5positioning5 libqt5sensors5 libqt5webkit5 qt5-image-formats-plugins qml-module-qtquick-controls libicu52 libpulse0 libjson-c-dev libexpat1-dev libdbus-1-3 libglib2.0-0 libkeyutils-dev libwrap0 libacl1 libreadline6 libbz2-1.0 libpcre3`
```

Comment:
I might remove this function if not using.  Sorry this above l was confused about what to fetch...then I realized that the packing Dockerfile only needs AppImageKit dependencies, since MuseScore dependencies were already copied.